schedule/yast/sle/guided_btrfs/guided_btrfs@s390x_kvm.yaml

```diff
@@ -0,0 +1,46 @@
+---
+name:           guided_btrfs
+description:    >
+  Guided Partitioning installation with btrfs filesystem.
+vars:
+  FILESYSTEM: btrfs
+  HDDSIZEGB: 40
+  YUI_REST_API: 1
+schedule:
+  default_systemd_target:
```

Comment:
we are missing the desktop here.

Reply:
Sorry. There is only [register_module_desktop](https://openqa.suse.de/tests/10061276/modules/register_module_desktop/steps/1/src) in the original test, no desktop related tests.

Reply:
Modifiled.